src/jormungandr/testing/jormungandr-integration-tests/src/jormungandr/explorer/block.rs

```diff
@@ -184,345 +130,88 @@ pub fn explorer_block_test() {
+        .as_bft_leader()
+        .build()
+        .start_node(temp_dir)
+        .unwrap();
 
     let params = ExplorerParams::new(BLOCK_QUERY_COMPLEXITY_LIMIT, BLOCK_QUERY_DEPTH_LIMIT, None);
     let explorer_process = jormungandr.explorer(params).unwrap();
-
-    let fragment_sender = FragmentSender::from_with_setup(
-        jormungandr.block0_configuration(),
-        FragmentSenderSetup::no_verify(),
-    );
-
-    let time_era = jormungandr.time_era();
-
-    let mut fragment_generator = FragmentGenerator::new(
-        sender,
-        receiver,
-        None,
-        jormungandr.to_remote(),
-        time_era.slots_per_epoch(),
-        2,
-        2,
-        2,
-        0,
-        fragment_sender,
-    );
-
-    fragment_generator.prepare(BlockDate::new(1, 0));
-
-    time::wait_for_epoch(2, jormungandr.rest());
-
-    let mem_checks: Vec<MemPoolCheck> = fragment_generator.send_all().unwrap();
-    FragmentVerifier::wait_and_verify_all_are_in_block(
-        Duration::from_secs(2),
-        mem_checks.clone(),
-        &jormungandr,
-    )
-    .unwrap();
-
-    let fragments_log = jcli.rest().v0().message().logs(jormungandr.rest_uri());
-    let fragment_log = fragments_log
-        .iter()
-        .find(|x| {
-            *x.fragment_id().to_string() == mem_checks.last().unwrap().fragment_id().to_string()
-        })
-        .unwrap();
-
-    let fragment_block_id =
-        if let &FragmentStatus::InABlock { date: _, block } = fragment_log.status() {
-            block
-        } else {
-            panic!("Fragment not in block")
-        };
-
-    let encoded_block = jcli
-        .rest()
-        .v0()
-        .block()
-        .get(fragment_block_id.to_string(), jormungandr.rest_uri());
-
-    let decoded_block = ExplorerVerifier::decode_block(encoded_block);
-
-    time::wait_for_epoch(3, jormungandr.rest());
-
-    explorer_process.wait_to_be_up(2, 20);
     let explorer = explorer_process.client();
 
-    let explorer_block_response = explorer
-        .blocks_at_chain_length(decoded_block.header().chain_length().into())
-        .unwrap();
-
-    assert!(
-        explorer_block_response.errors.is_none(),
-        "{:?}",
-        explorer_block_response.errors.unwrap()
-    );
-
-    let explorer_blocks_data = explorer_block_response.data.unwrap();
-    let explorer_block = explorer_blocks_data.blocks_by_chain_length.first().unwrap();
-
-    ExplorerVerifier::assert_block_by_chain_length(decoded_block, explorer_block).unwrap();
+    for (incorrect_block_id, error_message) in incorrect_block_ids {
+        let response = explorer.block_by_id(incorrect_block_id.to_string());
+        assert!(response.as_ref().unwrap().errors.is_some());
+        assert!(response.as_ref().unwrap().data.is_none());
+        assert!(response
+            .unwrap()
+            .errors
+            .unwrap()
+            .first()
+            .unwrap()
+            .message
+            .contains(error_message));
+    }
 }
```

Comment:
I think in this file all_block_test and block_by_chain_lenght test are gone missing